ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml

```diff
@@ -36,6 +36,22 @@ tests:
     - chain: openshift-qe-ocp-qe-perfscale-ci-tests
     workflow: rosa-aws-sts
   timeout: 6h0m0s
+- as: perfscale-rosa-multiaz-data-path-9nodes-periodic
+  cron: 0 13 * * 2
+  steps:
+    cluster_profile: aws-qe
```

Comment:
think we want tot use _aws-perfscale-qe_ profile here

Reply:
You are right anything <24 nodes will use `aws-perfscale-qe`